platform/linux-generic/include/odp_config_internal.h

```diff
@@ -65,6 +65,15 @@
 #define CONFIG_PACKET_HEADROOM 128
 
 /*
+ * Maximum packet headroom
+ *
+ * This defines the maximum number of headroom bytes that newly created packets
+ * can be configured. This value applies to both ODP packet input and user
+ * allocated packets.
+ */
+#define CONFIG_PACKET_MAX_HEADROOM 258
```

Comment:
Any particular reason for this "magic number"?

Reply:
We had a requirement from a customer for 256B headroom and I am not aware of any request bigger than this hence this magic number.

Reply:
Ok, but odp-linux really has no built-in maximum for this so we shouldn't impose one artificially. The effective max_headroom would be something like `CONFIG_PACKET_MAX_SEG_LEN - 1`.

Reply:
Okay